.idea/vcs.xml

```diff
@@ -2,5 +2,6 @@
 <project version="4">
```

Comment:
can you remove this file from the git project? this file is not needed on the git repo

Reply:
The file is now deleted